chimoney.md

```diff
@@ -0,0 +1,187 @@
+# Contributing to Chimoney Projects: A Hacktoberfest 2024 Guide
+
+## 1. Introduction to Hacktoberfest and Chimoney
+
+### What is Hacktoberfest?
+
+Hacktoberfest is an annual event that celebrates open source software and encourages meaningful contributions to open source projects. It's sponsored by DigitalOcean and runs throughout the month of October. Participants who successfully complete the challenge by making a certain number of valid pull requests to open source repositories are eligible for exciting rewards and recognition.
```

Comment:
Hi @Danbaba1, for this part `making a certain number` the number is 4 pull requests, please add that for clarity